backend/getData.js

```diff
@@ -27,7 +27,11 @@ function getData(element: Object): DataType {
   var text = null;
   var publicInstance = null;
   var nodeType = 'Native';
-  if (element._currentElement === null || element._currentElement === false) {
+  // Sometimes element is just a string or number
+  if (typeof element !== "object") {
```

Comment:
`_renderedChildren` should be null: http://react.jsbin.com/dikerovequ/edit?js,output.